templates/tests/server-test.yaml

```
@@ -7,6 +7,7 @@ metadata:
   annotations:
     "helm.sh/hook": test
 spec:
+  {{- include "imagePullSecrets" . | nindent 6 }}
```

I think the indentation just needs to be a little less here:

```suggestion
  {{- include "imagePullSecrets" . | nindent 2 }}
```